_posts/2021-12-16-jaas-realm.adoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: 'Using JAAS realm in Elytron'
+date: 2021-12-16
+tags: elytron wildfly JAAS security realm
+synopsis: An example of JAAS realm usage in WildFly.
+author: dvilkola
+---
+
+Elytron subsystem already provides various security realms like LDAP realm, JDBC realm, filesystem realm, and others for common use cases. Starting from WildFly 26, you can also configure JAAS security realm in Elytron subsystem in order to use custom Login Modules for authentication and authorization.
```

Comment:
Just a minor comment, s/Elytron subsystem/The Elytron subsystem

Reply:
s/JAAS security realm/JAAS security realms

Reply:
Fixed.